libs/core/src/lib/select/select.component.ts

```diff
@@ -769,7 +769,10 @@ export class SelectComponent implements
                 this.selectOptionsListTemplate,
                 SelectMobileComponent,
                 { container: this._elementRef.nativeElement },
-                { injector: Injector.create({ providers: [{ provide: SELECT_COMPONENT, useValue: this }] }) }
+                { 
+                    injector: Injector.create({ providers: [{ provide: SELECT_COMPONENT, useValue: this }] }),
+                    services: [this._rtlService]
```

Comment:
please don't use `services` option, this behaviour is implicit, instead we can use injector property and define there everything we need.

Reply:
fixed